iOS internship/ForecastViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ForecastViewController.swift
+//  iOS internship
+//
+//  Created by pavels.vetlugins on 26/09/2018.
+//  Copyright © 2018 pavels.vetlugins. All rights reserved.
+//
+
+import UIKit
+
+class ForecastViewController: UIViewController, GetForecastDelegate {
+    
+    
+    @IBOutlet weak var titleBar: UINavigationItem!
```

Comment:
Make these `private`, there should be no access to these outside view controller.